examples/client_example/client_example.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python
+"""Python Client Example."""
+
+# Copyright 2012 - 2017, New York University and the TUF contributors
+# SPDX-License-Identifier: MIT OR Apache-2.0
+
+import argparse
+import logging
+import os
+import shutil
+import sys
+from pathlib import Path
+
+from tuf.exceptions import RepositoryError
+from tuf.ngclient import Updater
+
+# define directory constants
+HOME_DIR = Path.home()  # user home dir
+DOWNLOAD_DIR = "./downloads"  # download dir
+METADATA_DIR = f"{HOME_DIR}/.local/share/tuf_metadata_example"  # metadata dir
```

Comment:
the directory name should be the app name so maybe 
`METADATA_DIR = f"{HOME_DIR}/.local/share/python-tuf-client-example"
`?

Also HOME_DIR seems to be defined just for this -- maybe the variable is not needed?